packages/block-editor/src/hooks/style.js

```diff
@@ -85,21 +86,35 @@ export function getInlineStyles( styles = {} ) {
 			// option and backwards compatibility for border radius support.
 			const styleValue = get( styles, path );
 
-			if ( !! subPaths && ! isString( styleValue ) ) {
-				Object.entries( subPaths ).forEach( ( entry ) => {
-					const [ name, subPath ] = entry;
-					const value = get( styleValue, [ subPath ] );
-
-					if ( value ) {
-						output[ name ] = compileStyleValue( value );
-					}
-				} );
-			} else if ( ! ignoredStyles.includes( path.join( '.' ) ) ) {
-				output[ propKey ] = compileStyleValue( get( styles, path ) );
+			if ( ! styleValue.useEngine ) {
```

Comment:
I think this should be `if ( ! STYLE_PROPERTY[ propKey ].useEngine ) {` ?

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/37978/commits/e1e231db3a2efea5ba4ebf80ffb41a3d4b68f22b